sample/src/main/res/layout/activity_test.xml

```diff
@@ -178,8 +183,8 @@
       android:layout_width="wrap_content"
       android:layout_height="match_parent"
       android:layout_gravity="start"
-      android:fitsSystemWindows="true"
-      android:background="@color/background_color">
+      android:background="@color/background_color"
+      android:fitsSystemWindows="true">
```

Comment:
Does reordering make any sense?

Reply:
It's just the default Code-Style in Android Studio.